CityOfDetroit/bloom
# 1434/listing finder q2

# Pull Request Template

## Issue Overview

This PR addresses #1434 


## Description

This PR refines the way the form data is handled based on if users are selecting previous, next, or skip to listings. Also, this PR includes an empty disclaimer page as a placeholder since it helped to emulate the data saving logic that the complete form requires.
 
## How Can This Be Tested/Reviewed?

This can be tested by going to the Listings Finder from the homepage by selecting "Find Listings for You". Then click through the form using next and previous to observe how selections persist after selecting next on a question, but will not if previous is clicked after the selections are made. Additionally, ensure that clicking next after the second question or the skip link takes you to the placeholder page where submit will display the listings page with the selected filters.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
